### PR TITLE
Docs https redirect example

### PR DESCRIPTION
**Pull request by mitchdraft**
https redirect docs
https://github.com/solo-io/gloo/issues/1908

### CODE REVIEW

**Comment by mitchdraft** [comment]
image

File: docs/content/gloo_routing/virtual_services/https_redirect.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: HTTPS Redirect
+weight: 4
+description: Route HTTP traffic to HTTPS services
+---
+
+To help users or old services find your https endpoints it is a common practice to redirect http traffic to https endpoints.
```
Comment (EItanya) [inline docs/content/gloo_routing/virtual_services/https_redirect.md]:
```suggestion
To help users, or old services, find your https endpoints, it is a common practice to redirect http traffic to https endpoints.
```

**Pull request merged**